examples/convert_i3_to_parquet.py

```diff
@@ -0,0 +1,74 @@
+"""Minimum working example (MWE) to use ParquetDataConverter."""
+
+import logging
+import os
+
+from graphnet.utilities.logging import get_logger
+
+from graphnet.data.i3extractor import (
+    I3FeatureExtractorIceCube86,
+    I3FeatureExtractorIceCubeUpgrade,
+    I3RetroExtractor,
+    I3TruthExtractor,
+)
+from graphnet.data.parquet.parquet_dataconverter import ParquetDataConverter
+
+logger = get_logger(level=logging.INFO)
+
+
+def main_icecube86():
+    """Main script function."""
+    paths = [
+        "/groups/icecube/asogaard/data/i3/i3_to_sqlite_workshop_test/level7_v02.00"
+    ]
+    pulsemap = "SRTInIcePulses"
+    gcd_rescue = "resources/GeoCalibDetectorStatus_AVG_55697-57531_PASS2_SPE_withScaledNoise.i3.gz"
+    outdir = "/groups/icecube/asogaard/temp/parquet_test_ic86"
+
+    converter = ParquetDataConverter(
```

Comment:
Would be really cool, and probably preferable, if we could create an extractor `extract_everything_in_frame` that would just loop over frames. It would make the user experience a lot nicer.